CHANGELOG.D/78.feature

```diff
@@ -1 +0,0 @@
-Add `towncrier` for release notes management
```

Comment:
@anayden, shouldn't these files be removed automatically after executing `make changelog`?